build/gen-windows-versioninfo.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+# Run goversioninfo to generate the resource.syso to embed version info.
+set -eux
+
+NAME="One Policy Agent (OPA)"
+VERSION=$(./build/get-build-version.sh)
+FLAGS=()
+
+# If building for arm64, then include the extra flags required.
+if [ -n "${1+x}" ] && [ "$1" = "arm64" ]; then
+    FLAGS=(-arm -64)
+fi
+
+goversioninfo "${FLAGS[@]}" \
```

Comment:
```suggestion

if ! command -v goversioninfo &> /dev/null; then
    # If goversioninfo isn't on the path, print an error message
    echo "Error: goversioninfo command not found" >&2
    exit 1
fi

goversioninfo "${FLAGS[@]}" \
```
[suggestion]: Since we're already using Bash for the script, adding small preflight check for `goversioninfo` being on the `PATH` might be helpful for debugging.

Reply:
Good idea, thanks!